google/gve/gve_rx.c

```diff
@@ -545,6 +545,8 @@ bool gve_rx_work_pending(struct gve_rx_ring *rx)
 	next_idx = rx->cnt & rx->mask;
 	desc = rx->desc.desc_ring + next_idx;
 
+	/* make sure we have synchronized the seq no with the device */
+	smp_mb();
```

Comment:
Did we ever confirm whether or not this was necessary? Or are we just adding it back until we upstream the removal of it?

Reply:
No, we have not confirmed that this needs to be put back. It's only being put back to match upstream till we take it out.